<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>568</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>849</width>
    <height>568</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>849</width>
    <height>568</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FunctionPlotter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="Plot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>681</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FunctionLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>681</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Write you function here:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PlotBtn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="MinSpin">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min limit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max limit</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="MaxSpin">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorLabel">
    <property name="geometry">
     <rect>
      <x>358</x>
      <y>0</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
